AppAuth-JS/built/node_support/index.d.ts

```diff
@@ -0,0 +1,3 @@
+export * from './crypto_utils';
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork